src/install.js

```diff
@@ -37,18 +53,36 @@ async function updateBrowsers(opts = {}) {
   }
 }
 
-function getPlaywrightBrowserPath(kind) {
+function getPlatformKey(browserName) {
+  const key = `${process.platform}:${browserName}`;
+  switch (key) {
+    case "darwin:firefox":
+    case "linux:firefox":
+    case "linux:chromium":
+      return key;
+  }
+
+  return undefined;
+}
+
+function getExecutablePath(runner, browserName) {
+  // Override with replay specific browsers.
   const replayDir = getDirectory();
 
-  switch (`${process.platform}:${kind}`) {
-    case "darwin:gecko":
-      return path.join(replayDir, "playwright", "firefox", "Nightly.app", "Contents", "MacOS", "firefox");
-    case "linux:gecko":
-      return path.join(replayDir, "playwright", "firefox", "firefox");
-    case "linux:chromium":
-      return path.join(replayDir, "playwright", "chrome-linux", "chrome");
+  const key = getPlatformKey(browserName);
+  if (!key) {
+    return null;
   }
-  return null;
+
+  return path.join(replayDir, runner, ...EXECUTABLE_PATHS[key]);
+}
+
+function getPlaywrightBrowserPath(kind) {
+  return getExecutablePath("playwright", kind);
+}
+
+function getPuppeteerBrowserPath(kind) {
+  return getExecutablePath("puppeteer", kind);
 }
```

Comment:
The more that i'm seeing the test integrations in the CLI, the more i think it might make sense for us to suggest that projects install the cli and then use it to setup the test.